test/orderbookReserve.js

```diff
@@ -3193,6 +3193,62 @@ contract('OrderbookReserve', async (accounts) => {
             assert.equal(orderList.length, 4);
         })
 
+        it("test can't 'revive' canceled (tok to eth) order by updating it", async() => {
+            let tokenWeiDepositAmount = new BigNumber(500 * 10 ** 18);
+            let kncTweiDepositAmount = 600 * 10 ** 18;
+            let ethWeiDepositAmount = (new BigNumber(0 * 10 ** 18)).add(30000);
+            await makerDeposit(maker1, ethWeiDepositAmount, tokenWeiDepositAmount, kncTweiDepositAmount);
+
+            let orderSrcAmountTwei = new BigNumber(9 * 10 ** 18);
+            let orderDstWei = new BigNumber(2 * 10 ** 18);
+
+            await reserve.submitTokenToEthOrder(orderSrcAmountTwei, orderDstWei, {from: maker1});
+            let rc = await reserve.submitTokenToEthOrder(orderSrcAmountTwei, orderDstWei.add(100), {from: maker1});
+            let id2ndOrder = rc.logs[0].args.orderId.valueOf();
+            rc = await reserve.submitTokenToEthOrder(orderSrcAmountTwei, orderDstWei.add(200), {from: maker1});
+            let id3rdOrder = rc.logs[0].args.orderId.valueOf();
+            await reserve.submitTokenToEthOrder(orderSrcAmountTwei, orderDstWei.add(300), {from: maker1});
+
+            await reserve.cancelTokenToEthOrder(id2ndOrder, {from: maker1});
+
+            let list = await reserve.getTokenToEthOrderList();
```

Comment:
Nit: no need to save this id as a variable as you do not use it later.

Reply:
I do use it. 
see below